apps/gitness/src/App.tsx

```diff
@@ -1,10 +1,11 @@
 import { I18nextProvider } from 'react-i18next'
-import { createBrowserRouter, RouterProvider } from 'react-router-dom'
+import { createBrowserRouter, Link, NavLink, Outlet, RouterProvider } from 'react-router-dom'
 
 import { QueryClientProvider } from '@tanstack/react-query'
 
 import { CodeServiceAPIClient } from '@harnessio/code-service-client'
 import { ToastProvider, TooltipProvider } from '@harnessio/ui/components'
+import { RouterProvider as RouterProviderV1 } from '@harnessio/ui/context'
```

Comment:
why is named with a `V1`?

Reply:
This is just temporary, eventually this will replace `RouterProvider` from `react-router-dom`, will rename it then.